### PR TITLE
Repeated key property fix

### LINKED ISSUE #1: KeyPropertiesField data property is a Model object
Hi,

Not sure if this is an issue or intended behavior.  It seems that KeyPropertiesField should be converting form data into Key objects (not Model objects). This is also what the docstring of ModelConverter indicates. 

In any case, it would be nice if this were an option at least. I will probably fork and work on it myself, but I was hoping to get your thoughts first.

Thanks,

Stephen

### PR DESCRIPTION
**Pull request by dhstack**
Fixes #1 and #6. Integrated #8 PR and fixes from https://github.com/xlevus/wtforms-ndb.

### CODE REVIEW

File: wtforms_appengine/__init__.py [hunk 1]
```diff
@@ -1 +1,5 @@
 __version__ = '0.1.1dev'
+
+from .ndb import model_form, ModelConverter
```
Comment (mikelambert) [inline wtforms_appengine/__init__.py]: I'm unclear why you needed to make these changes to the __init__.py?

Comment (dhstack) [inline wtforms_appengine/__init__.py]: It looks like these lines were added in by @xlevus so you'd have to ask him. I just merged his entire branch into this PR and this was part of it. I assumed it helped with the functionality of his work.

**Comment by mikelambert** [comment]
So @dhstack, you grabbed from https://github.com/xlevus/wtforms-ndb/commits/master, right? Looks like there's been another dozen commits since you branched it early last year, so ideally I'd be merging some mix of https://github.com/xlevus/wtforms-ndb/commits/master and the changes you made after merging to get it working back within wtforms_appengine.

@dhstack not sure if you're still actively working with wtforms and have any interest in this? Or better yet, @xlevus, not sure if you have any interest in merging your improvements back into wtforms-appengine package and un-forking things? I just recently volunteered to maintain wtforms-appengine, and so have been going through trying to process all the ancient pull requests, hoping I'm not too late... :)

**Comment by xlevus** [comment]
I'll try and find some time to submit a PR to wtforms-appengine over the
next few days. No promises ;)

**Comment by dhstack** [comment]
@mikelambert Yes, you are correct. I grabbed from https://github.com/xlevus/wtforms-ndb/commits/master a while ago so it looks like you'll need to merge all the changes from there and what I did.

I'm not working with wtforms anymore - moved on to other projects. Best of luck getting everything working :)

**Comment by mikelambert** [comment]
I'm going to close this one as I just merged in xlevus's other pull request with all the requisite fixes.

**Pull request closed without merging**